### DISCUSSION TITLE
V3 Support?

### DISCUSSION THREAD

**Discussion by mstfkhazaal** [andrewdwallo/filament-companies#76] [Feature requests]
Please support filament v3

**Comment by andrewdwallo** [comment]
@mstfkhazaal I would be more open to it if I had contributors help me with it.

**Reply by mstfkhazaal** [reply]
I want to help with that, I'll try but I don't have enough information to do that

**Reply by andrewdwallo** [reply]
What information do you need?

**Reply by mstfkhazaal** [reply]
How to make a plugin to the Filament I have not tried before I'll read the docs on the topic
There are several questions in my mind. Many features were added in version 3, including those supported by this add-on. Do you want to delete them from this plugin?
like MultiTenancy UI

**Reply by andrewdwallo** [reply]
@mstfkhazaal That's a good question. I thought about it and I think potentially removing methods and features that aren't unique to my package anymore and keeping what is unique...

**Comment by andrewdwallo** [comment]
@mstfkhazaal V3 support coming soon! You can test the 3.x branch.
